validator.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'constants'
+
+module Validator
+  
+  def self.validateInput(inputItems)
```

Comment:
Use snake casing

Reply:
Yup. Ruby is heavily focussed on snake casing. Will keep in mind.